client/src/components/TagColumn/TagColumn.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './TagColumn.scss';
+import Tags from '../Tags/Tags';
+
+/**
+ * Presentational component for a column of Tags
+ */
+export default class TagColumn extends React.Component {
```

Comment:
Why is this component necessary?

Reply:
When I first started I thought it'd be more, but it's not. I'll remove it.